ruma-federation-api/src/keys/claim_keys/v1.rs

```diff
@@ -7,8 +7,10 @@ use ruma_api::ruma_api;
 use ruma_identifiers::{DeviceId, DeviceKeyAlgorithm, DeviceKeyId, UserId};
 use serde::{Deserialize, Serialize};
 
-type KeyClaim = BTreeMap<UserId, BTreeMap<Box<DeviceId>, DeviceKeyAlgorithm>>;
-type KeyResponse = BTreeMap<UserId, BTreeMap<Box<DeviceId>, BTreeMap<DeviceKeyId, KeyObject>>>;
+/// A claim for one time keys
+pub type OneTimeKeyClaim = BTreeMap<UserId, BTreeMap<Box<DeviceId>, DeviceKeyAlgorithm>>;
```

Comment:
Type name sounds like it's only for one key. Also, could you move these beow the `ruma_api!` invocation actually (and separate them with an empty line)? We don't usually declare anything before that.

```suggestion
pub type OneTimeKeyClaims = BTreeMap<UserId, BTreeMap<Box<DeviceId>, DeviceKeyAlgorithm>>;
```